rbac/onboarding-proxy-sa.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
You should create this file under config/rbac and add it to Kustomization.yml, name should be ux_backend_sa.yaml not the proxy.

I believe if you delete this file and generate bundle this'll not get created unless you add to Kustomization.yml

Reply:
I discussed it with leela over DM and we are keeping it here and removing --extra-service-account

Reply:
i can rename the file as it make sense

Reply:
The server should not run with the same service account as OCS operator. if we remove this, which service account should we use instead?

Reply:
The premise is if we are mentioning a serviceaccount in csv then olm will create and manage that serviceaccount implictly - this seems to be how ocs operator does

(or)

if we have `--extra-service-account` arg, an explicit file needs to be added and olm will just deploy that sa - this is how client operator does

But, if we have an explicit sa file with no arg supplied to bundle target, then current csv owns sa and during upgrade new csv also tries to own existing sa which is already being owned by old csv and get stuck. I remember seeing this scenario during deployer debugging.

I'm so sry for holding this up, I want to hear that the deployment is happening as expected when we are testing it properly.

Reply:
@rchikatw my initial comment is correct! Pls have role, binding and sa files in config/rbac and use extra-sa arg as well.